### PR TITLE
ci/docs: enable not fetch assets

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Enable developing locally without overwriting links in pages...

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

cc @borda @carmocca @justusschock @awaelchli

### CODE REVIEW

File: docs/source-pytorch/conf.py [hunk 1]
```diff
@@ -28,6 +28,8 @@
 # VARIABLES WHEN WORKING ON DOCS... MAKE THIS TRUE TO BUILD FASTER
 # -----------------------
 _PL_FAST_DOCS_DEV = bool(int(os.getenv("PL_FAST_DOCS_DEV", 0)))
+_DOCS_FETCHING_ASSETS = int(os.environ.get("DOCS_FETCHING_ASSETS", True))
```
Comment (carmocca) [inline docs/source-pytorch/conf.py:31]: Why not include this under FAST_DOCS_DEV? Do we need one more flag?

Comment (Borda) [reply docs/source-pytorch/conf.py:31]: I would keep it separate as it does different things...

Comment (Borda) [reply docs/source-pytorch/conf.py]: ok, reverted...

**Review by carmocca — APPROVED** [review]
It would be nice to also have FAST_DOCS_DEV on App and Fabric. I always forget which flag name I need to enable

**Pull request merged**